x-pack/plugins/uptime/public/components/functional/status_bar.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiHealth, EuiLink, EuiPanel } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import moment from 'moment';
+import React from 'react';
+
+interface Props {
+  duration?: string;
+  host?: string;
+  port?: string;
+  scheme?: string;
+  status?: string;
+  timestamp?: string;
+}
+
+export const StatusBar = ({ timestamp, host, port, duration, scheme, status }: Props) => (
+  <EuiPanel>
+    <EuiFlexGroup gutterSize="l">
+      <EuiFlexItem grow={false}>
+        <EuiHealth
+          aria-label={i18n.translate('xpack.uptime.monitorStatusBar.healthStatusMessageAriaLabel', {
+            defaultMessage: 'Monitor status',
+          })}
+          color={status === 'up' ? 'success' : 'danger'}
+          style={{ lineHeight: 'inherit' }}
+        >
+          {status === 'up'
+            ? i18n.translate('xpack.uptime.monitorStatusBar.healthStatusMessage.upLabel', {
+                defaultMessage: 'Up',
+              })
+            : i18n.translate('xpack.uptime.monitorStatusBar.healthStatusMessage.downLabel', {
+                defaultMessage: 'Down',
+              })}
+        </EuiHealth>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <FormattedMessage
+          id="xpack.uptime.monitorStatusBar.healthStatus.hostMessage"
+          defaultMessage="Host: {host}"
```

Comment:
For me this shows up broken:

![image](https://user-images.githubusercontent.com/131427/51709975-790b3a80-1fed-11e9-9f2f-092629c5b2ee.png)